process/block/metrics.go

```diff
@@ -126,6 +126,14 @@ func incrementCountAcceptedBlocks(
 	}
 
 	bitMap := header.GetPubKeysBitmap()
+	indexOutOfBounds := myIndex/8 >= len(bitMap)
```

Comment:
the big question here is: why this could happen in a normal case?

Reply:
normally it shouldn't happen. but there was a spam which had hardcoded fields of the block which somehow passed the interceptor